Doc/library/inspect.rst

```diff
@@ -153,6 +153,17 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 | f_trace           | tracing function for this |
 |                 |                   | frame, or ``None``        |
 +-----------------+-------------------+---------------------------+
+|                 | f_trace_lines     | is a tracing event        |
```

Comment:
I think it's better not to use questions and use one of the following formulations:

- indicate whether a tracing event is triggered for each source line
- true if a tracing event is triggered for each source line, false otherwise